applications/directory/src/components/Toolbar.js

```diff
@@ -15,7 +15,7 @@ function Toolbar() {
           '& li': {
             margin: '0.625rem 0',
             width: 'auto',
-            padding: '0.5rem 0',
+            padding: ['0.5rem 1rem', '0.5rem 1.5rem'],
             '& a': {
```

Comment:
Need to move this styling to the `Link` element below.

Reply:
done